config/database.yml

```diff
@@ -5,7 +5,9 @@ default: &default
   port: <%= ENV.fetch('POSTGRES_PORT', '5432') %>
    # ref: https://github.com/mperham/sidekiq/issues/2985#issuecomment-531097962
   pool: <%= Sidekiq.server? ? Sidekiq.options[:concurrency] : ENV.fetch('RAILS_MAX_THREADS', 5) %>
-
+  variables:
+    statement_timeout: <%= ENV["STATEMENT_TIMEOUT"] || "5s" %>
```

Comment:
You need to update the `env.example` file for this it says`POSTGRES_STATEMENT_TIMEOUT`

Reply:
Nice catch. we intend to use `POSTGRES_STATEMENT_TIMEOUT` .will update it here.